.gitignore

```diff
@@ -1,4 +1,23 @@
-*.swp
-build
-node_modules
-translated
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Licence is still missing.

Reply:
I see that this is a general problem. On the current `master` there is also no LICENCE. We have to add that. I'll make a story for it.